examples/python/example1.py

```
@@ -0,0 +1,6 @@
+def add(a,b):
+    return a+b
+
+def process_batch(batch):
+  print(add(1,1))
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 4

(W0311)